choose-thumbnail/index.md

```diff
@@ -0,0 +1,19 @@
+## Choosing a thumbnail image
```

Comment:
let's combine this with "What is a thumbnail" it doesn't make sense to have 3 different help questions for the topic of thumbnails #24